docs/pages/components/TagButton.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: TagButton
+description: Component that is an extension of the existing <Button variant="tag"> that requires those properties listed in the Props Section
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/TagButton/index.tsx
+---
+
+import { TagButton } from '@deque/cauldron-react'
+
+```js
+import { TagButton } from '@deque/cauldron-react'
+```
+
+## Example
+
+```jsx example
+<div style={{ display: "flex", gap: "8px" }}>
+  <TagButton label={"Label"} value={"value"} action={() => console.log("test")} icon={"pencil"} />
+
+  <TagButton label={"Label"} value={"value"} action={() => console.log("test")} icon={"pencil"} disabled={true} />
+</div>
+```
+
+
+## Props
+
+### TagButton
+
+<ComponentProps 
+  className={true} 
+  props={[
+    {
+      name: 'label',
+      required: true,
+      type: ['string', 'number', 'ReactElement', 'ReactFragment', 'ReactPortal'],
+      description: 'Label for the tag button.'
+    },
+    {
+      name: 'value',
+      required: true,
+      type: ['string', 'number', 'ReactElement', 'ReactFragment', 'ReactPortal'],
+      description: 'Value for the tag button.'
+    },
+    {
+      name: 'action',
+      required: true,
+      type: '(e: React.MouseEvent<HTMLButtonElement>) => void',
+      description: 'Label for the tag button.'
+    },
+    {
+      name: 'icon',
+      type: 'string',
+      required: true,
+      description: 'Icon to use for the TagButton'
+    },
+    {
+      name: 'disabled',
+      required: false,
+      type: 'boolean',
+      description: 'Disabled state of the element'
+    },
+  ]}
+/>
```

Comment:
We should include "related components" here, and also cross-link with those related components. I think `Button` and `Tag` would suffice.